src/sentry/incidents/serializers/alert_rule.py

```diff
@@ -94,6 +95,10 @@ class AlertRuleSerializer(CamelSnakeModelSerializer[AlertRule]):
     activation_condition = serializers.IntegerField(required=False, allow_null=True, min_value=0)
     description = serializers.CharField(required=False, allow_blank=True)
 
+    sensitivity = serializers.CharField(required=False, allow_null=True)
+    seasonality = serializers.CharField(required=False, allow_null=True)
+    detection_type = serializers.CharField(required=False, default=AlertRuleDetectionType.STATIC)
```

Comment:
Should this be required? Or is this shared by the PUT endpoint and can't be for that reason?

Reply:
I followed the convention set by the owner field (set required=True once it starts being sent by the frontend)